units/host_stats.go

```diff
@@ -109,4 +109,13 @@ func (j *hostStatsJob) Run(_ context.Context) {
 			})
 		}
 	}
+
+	count, err := host.CountVirtualWorkstationsByInstanceType()
```

Comment:
do we want this to run at this frequency?

Reply:
It runs every 15 minutes. I put it here since it's another host stats job. We can use this to graph change over time.